test/runner/lib/cloud/vcenter.py

```diff
@@ -139,8 +183,63 @@ def _get_simulator_address(self):
         ipaddress = results[0]['NetworkSettings']['IPAddress']
         return ipaddress
 
+    def _setup_dynamic_baremetal(self):
+        """Request Esxi credentials through the Ansible Core CI service."""
+        display.info('Provisioning %s cloud environment.' % self.platform,
+                     verbosity=1)
+
+        config = self._read_config_template()
+
+        aci = self._create_ansible_core_ci()
+
+        if not self.args.explain:
+            response = aci.start()
+            values = response['esxi']
+            self.aci = aci
+
+            config = self._populate_config_template(config, values)
+            self._write_config(config)
+
+    def _create_ansible_core_ci(self):
+        """
+        :rtype: AnsibleCoreCI
+        """
+        return AnsibleCoreCI(self.args, 'vmware', 'vmware',
+                             persist=False, stage=self.args.remote_stage,
+                             provider='vmware')
+
     def _setup_static(self):
-        raise NotImplementedError()
+        parser = ConfigParser()
+        parser.read(self.config_static_path)
+
+        self.endpoint = parser.get('DEFAULT', 'vcenter_hostname')
+        self.port = parser.get('DEFAULT', 'vcenter_port')
+
+        if parser.get('DEFAULT', 'vmware_validate_certs') == 'no':
```

Comment:
I suggest:
 if parser.get('DEFAULT', 'vmware_validate_certs').lower() in ('no', 'false'):